rpcd/playbooks/roles/rpc_pre_upgrade/templates/status.txt.j2

```diff
@@ -25,6 +25,11 @@ cinder service-list output as at {{ ansible_date_time.date }}-{{ ansible_date_ti
 {{ line }}
 {% endfor %}
 
+orchestration service-list output as at {{ ansible_date_time.date }}-{{ ansible_date_time.time }}
```

Comment:
I appreciate it's not running the command but I believe it is supposed to be showing the command run to get the output. `nova service-list` and `cinder service-list` are both are run by `rpcd/playbooks/roles/rpc_pre_upgrade/tasks/gather_stats.yml`. To my knowledge there is no client called orchestration and `orchestration service-list` is not run by any task.